static/cmd/main.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/status-im/status-go/geth/common/cipher"
+	"gopkg.in/urfave/cli.v1"
+)
+
+/* Examples:
```

Comment:
Minor: it could be put before `package main` following https://blog.golang.org/godoc-documenting-go-code.